### PR TITLE
[wpmlbridge-94] Replace old WPML autoloader with simple require

### PR DESCRIPTION
**Pull request by strategio**
I took the opportunity to adjust the unit tests. I removed some unrelated tests and adjusted the existing ones to run with `WP_Mock`.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+#1.0.1
```
Comment (andreasciamanna) [inline changelog.md]: @strategio  Since the type of change, I think is worth pushing to 1.1.0.

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "wpml/mu-domain-mapping",
+  "description": "WPML ST Dependencies",
```
Comment (andreasciamanna) [inline composer.json]: @strategio  maybe change this description?

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "wpml/mu-domain-mapping",
```
Comment (andreasciamanna) [inline composer.json]: @strategio please use "otgs/wpml-mu-domain-mapping"

File: plugin.php [hunk 4]
```diff
@@ -5,21 +5,24 @@
 Description: This is a bridge plugin to make <a href="https://wordpress.org/plugins/wordpress-mu-domain-mapping/" target="_blank">MU Domain Mapping</a> compatible with WPML
 Author: OnTheGoSystems
 Author URI: http://www.onthegosystems.com/
-Version: 1.0.0
+Version: 1.0.1
```
Comment (andreasciamanna) [inline plugin.php]: @strategio same here about the version number

File: plugin.php [hunk 5]
```diff
@@ -5,21 +5,24 @@
...
 require_once 'classes/class-wpml-mu-domain-mapping-requirements.php';
 new WPML_MU_Domain_Mapping_Requirements();
 
 function wpml_mu_domain_mapping_enable() {
 	global $wpdb, $sitepress;
 
-	$wpml_auto_loader_instance = WPML_Auto_Loader::get_instance();
-	$wpml_auto_loader_instance->register( dirname( __FILE__ ) . '/' );
-
+	require_once 'classes/class-wpml-mu-domain-mapping-filters.php';
```
Comment (andreasciamanna) [inline plugin.php:25]: @strategio Don't we use autoloading?

File: plugin.php [hunk 6]
```diff
@@ -5,21 +5,24 @@
...
 */
 
+if ( defined( 'WPML_MU_DOMAIN_MAPPING_PATH' ) || get_option( '_wpml_inactive' ) ) {
+	return;
+}
+
 define( 'WPML_MU_DOMAIN_MAPPING_ID', plugin_basename( __FILE__ ) );
+define( 'WPML_MU_DOMAIN_MAPPING_PATH', dirname( __FILE__ ) );
 
 require_once 'classes/class-wpml-mu-domain-mapping-requirements.php';
```
Comment (andreasciamanna) [inline plugin.php:19]: @strategio Don't we use autoloading?

**Comment by sciamannikoo** [comment]
@strategio this PR is impossible to review. 

199 changes: really?

Only to fix a very simple bug?

**Review by andreasciamanna — CHANGES_REQUESTED** [review]
@strategio this PR is impossible to review. 

199 changes: really?

Only to fix a very simple bug?

Please:
- Create a PR only for the specific issue and try to not get tempted to rewrite everything again
- If you want to refactor, write tests, move files around, each of these things should be part of a separated commit

File: plugin.php (hunk 5 above)
Comment (strategio) [reply plugin.php:25]: @sciamannikoo we agreed to avoid using composer for non-dev, so they can just download and install it on their site out-of-the-box.

**Comment by strategio** [comment]
@sciamannikoo, I changed the version to 1.1.0.

We have an old ticket *wpmlbridge-63* that I'll use to open a PR for the tests cleanup and update.

File: plugin.php (hunk 5 above)
Comment (andreasciamanna) [reply plugin.php:25]: Right

**Review by andreasciamanna — APPROVED** [review]
It looks good to me, now.

**Comment by dimitrismitsis** [comment]
@sciamannikoo Could you please merge this? I don't have write access :)

**Comment by strategio** [comment]
I will do this @dimitrismitsis

**Pull request merged**